source/_docs/installation/raspberry-pi.markdown

```diff
@@ -89,7 +89,7 @@ You can now reach your installation on your Raspberry Pi over the web interface
 
 <div class='note'>
 
-When you run the `hass` command for the first time, it will download, install and cache the necessary libraries/dependencies. This procedure may take anywhere between 5 to 10 minutes. During that time, you may get "site cannot be reached" error when accessing the web interface. This will only happen for the first time, and subsequent restarts will be much faster.
+When you run the `ha` command for the first time, it will download, install and cache the necessary libraries/dependencies. This procedure may take anywhere between 5 to 10 minutes. During that time, you may get "site cannot be reached" error when accessing the web interface. This will only happen for the first time, and subsequent restarts will be much faster.
```

Comment:
This change is incorrect. `ha` is an external CLI, this refers to running Home Assistant Core, `hass` is the correct command. This change needs to be reverted.